src/main/resources/i18n/strings.properties

```diff
@@ -124,6 +124,29 @@ unlock.error.message=Unable to unlock vault
 unlock.error.invalidMountPoint.notExisting=Mount point "%s" is not a directory, not empty or does not exist.
 unlock.error.invalidMountPoint.existing=Mount point "%s" already exists or parent folder is missing.
 unlock.error.invalidMountPoint.driveLetterOccupied=Drive Letter "%s" is already in use.
+## Hub
+### Waiting
+hub.auth.message=Waiting for authentication…
+hub.auth.description=You should automatically be redirected to the login page.
+hub.auth.loginLink=Not redirected? Click here to open it.
+### Receive Key
+hub.receive.message=Processing response…
+hub.receive.description=Cryptomator is receiving and processing the response from Hub. Please wait.
+### Register Device
+hub.register.message=Device unknown
+hub.register.description=Cryptomator Hub does not recognize this device. Please register it.
```

Comment:
```suggestion
hub.register.message=First access from this device
hub.register.description=Cryptomator Hub does not know this device yet. Please choose a name, so the vault owner knows which device requires access rights.
```

Reply:
Changed in https://github.com/cryptomator/cryptomator/pull/2319/commits/73442cddc8cc2c529c66fa07898a671b75a08a7d

I changed the texts to make the intent of the dialog clearer, while keeping the necessary info, why the user has to do it, dense.